src/components/common/TreeExplorer.vue

```diff
@@ -82,14 +82,18 @@ const getTreeNodeIcon = (node: TreeExplorerNode) => {
 }
 const fillNodeInfo = (node: TreeExplorerNode): RenderedTreeExplorerNode => {
   const children = node.children?.map(fillNodeInfo)
+  const totalLeaves = node.leaf
+    ? 1
+    : children.reduce((acc, child) => acc + child.totalLeaves, 0)
   return {
     ...node,
     icon: getTreeNodeIcon(node),
     children,
     type: node.leaf ? 'node' : 'folder',
-    totalLeaves: node.leaf
-      ? 1
-      : children.reduce((acc, child) => acc + child.totalLeaves, 0)
+    totalLeaves,
+    badgeText: node.getBadgeText
+      ? node.getBadgeText(node)
+      : totalLeaves.toString()
```

Comment:
`totalLeaves.toString()` => `null`, as we are doing fallback in `TreeExplorerTreeNode`.

Reply:
good catch, fixed